application/root-app/src/main/AndroidManifest.xml

```diff
@@ -1,17 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    package="com.stepango.blockme.root.library">
+        xmlns:tools="http://schemas.android.com/tools">
 
     <uses-permission android:name="android.permission.INTERNET" />
 
     <application
-        android:name=".SampleApp"
-        android:fullBackupContent="true"
-        android:allowBackup="true"
-        android:extractNativeLibs="false"
-        android:icon="@mipmap/ic_launcher"
-        android:label="@string/app_name"
+            android:name=".SampleApp"
+            android:fullBackupContent="true"
+            android:allowBackup="true"
+            android:extractNativeLibs="false"
+            android:icon="@mipmap/ic_launcher"
+            android:label="@string/app_name"
```

Comment:
Some issues with formatting, I guess.

Reply:
yeah, something strange happened, thank you